plugins/keycloak-backend/package.json

```diff
@@ -14,7 +14,9 @@
     "role": "backend-plugin"
   },
   "scripts": {
-    "start": "backstage-cli package start",
+    "start": "concurrently -n 'keycloak,backstage' 'yarn start:keycloak' 'yarn start:dev' 'open http://localhost:8080/admin/master/console/#/janus-realm' 'open http://localhost:7007/catalog/entities'",
+    "start:dev": "backstage-cli package start",
+    "start:keycloak": "podman run -p 8080:8080 -e 'KEYCLOAK_ADMIN=admin' -e 'KEYCLOAK_ADMIN_PASSWORD=admin' -v ./__fixtures__/keycloak-realm.json:/opt/keycloak/data/import/keycloak-realm.json quay.io/keycloak/keycloak:22.0.1 start-dev --import-realm",
```

Comment:
Is it possible to use turbo instead of concurrently here? I think it should be possible.

Reply:
@schultzp2020 can you help me understand how it should work? I'm clueless atm. This is a single workspace specific task, that needs to spawn 2 commands instead of one. How is that achievable via turbo and what would be the benefit in this case? 🤷